packages/ui/src/lib/button/Button.stories.svelte

```diff
@@ -1,16 +1,51 @@
 <script>
-    import {Meta, Template, Story} from '@storybook/addon-svelte-csf';
+	import { ArrowCircleDownIcon } from '@rgossiaux/svelte-heroicons/solid';
 
-    import Button from "./Button.svelte";
+	import { Meta, Story, Template } from '@storybook/addon-svelte-csf';
 
+	import Button from './Button.svelte';
 </script>
 
-<Meta title="Ui/Button" component={Button}/>
+<Meta title="Ui/Button" component={Button} />
 
 <Template let:args>
-    <Button {...args}>
-        I'm a button from UI!
-    </Button>
+	<Button {...args}>I'm a button from UI!</Button>
 </Template>
 
 <Story name="Default" />
+
+<Story name="Primary">
+	<Button style="primary">Primary Style</Button>
+</Story>
+
+<Story name="Secondary">
+	<Button style="secondary">Secondary Style</Button>
+</Story>
+
+<Story name="Ghost">
+	<Button style="ghost">Ghost Style</Button>
+</Story>
+
+<Story name="Disabled" args={{ disabled: true }} />
+
+<Story name="Sizes">
+	<Button size="xs">xs</Button>
+	<Button size="sm">sm</Button>
+	<Button size="base">base</Button>
+	<Button size="lg">lg</Button>
+	<Button size="xl">xl</Button>
+</Story>
```

Comment:
```suggestion
<Story name="Sizes">
    <div class="flex items-end space-x-2">
        <Button size="xs">xs</Button>
        <Button size="sm">sm</Button>
        <Button size="base">base</Button>
        <Button size="lg">lg</Button>
        <Button size="xl">xl</Button>
    </div>
</Story>
```

Align the bottom of the buttons.